src/mon/PGMonitor.cc

```diff
@@ -1375,8 +1375,11 @@ void PGMonitor::dump_object_stat_sum(TextTable &tbl, Formatter *f,
     tbl << stringify(si_t(sum.num_bytes));
     int64_t kb_used = SHIFT_ROUND_UP(sum.num_bytes, 10);
     float used = 0.0;
-    if (pg_map.osd_sum.kb > 0)
-      used = (float)kb_used * raw_used_rate * curr_object_copies_rate / pg_map.osd_sum.kb;
+    if (avail) {
+      used = static_cast<float>(sum.num_bytes) / (sum.num_bytes + avail);
```

Comment:
I think you also need to include the raw_used_rate and curr_object_copies_rate to turn the logical usage into the replicated usage (1GB USED needs to use up 3GB of space with replication = 3).